### ISSUE TITLE
security issue: old node-pre-gyp version

### ISSUE THREAD

**Issue by rogierschouten** [mapbox/node-sqlite3#958]
**Solution**: use node-pre-gyp version 0.9.0 or later

**Problem**:

Run these commands:
```shell
npm install -g nsp
cd my/module/that/uses/sqlite3
nsp check
```

Output: security vulnerability
```
$ nsp check
(+) 1 vulnerability found
┌────────────┬────────────────────────────────────────────────────────────────────┐
│            │ Prototype pollution attack                                         │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Name       │ hoek                                                               │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ CVSS       │ 4 (Medium)                                                         │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Installed  │ 2.16.3                                                             │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Vulnerable │ <= 4.2.0 || >= 5.0.0 < 5.0.3                                       │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Patched    │ > 4.2.0 < 5.0.0 || >= 5.0.3                                        │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ Path       │ my-module > sqlite3@3.1.13 > node-pre-gyp@0.6.39 >     │
│            │ hawk@3.1.3 > hoek@2.16.3                                           │
├────────────┼────────────────────────────────────────────────────────────────────┤
│ More Info  │ https://nodesecurity.io/advisories/566                             │
└────────────┴────────────────────────────────────────────────────────────────────┘
```

**Comment by rogierschouten** [comment]
See also mapbox/node-pre-gyp/issues/346

**Comment by thom-nic** [comment]
I think this can be closed: sqlite3 v4.0.0 uses node-pre-gyp@0.9.0 and no longer depends on hoek.

**Comment by rogierschouten** [comment]
@thom-nic  yes you are right
